src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/PaymentAccountValidator.java

```diff
@@ -77,4 +78,27 @@ public void checkPbasAreUniqueWithOrgId(Set<String> paymentAccounts, String orgI
         }
     }
 
+    public boolean checkSinglePbaIsValid(String pbaAccount) {
+        if (pbaAccount.length() == 10) {
+            Pattern pattern = Pattern.compile("(?i)pba\\w{7}$");
+            Matcher matcher = pattern.matcher(pbaAccount);
+            if (matcher.matches()) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public boolean checkSinglePbaIsUnique(String pbaAccount) {
```

Comment:
will this call db as many times pba passed in request ? if it is we need to avoid db calls and use IN clause to find pba in one db call

Reply:
removed method